### DISCUSSION TITLE
[Mailer] Problem to sending email

### DISCUSSION THREAD

**Discussion by gourdina** [symfony/symfony#45316] [Q&A] [answered]
Hello,
I make it in controller :
$email = (new Email())
          ->from($from)
          ->to($to) ...
When $to= "xxx@gmail.com", sending email failed. 
MAILER_DSN=smtp://xxx@orange.fr:****@smtp.orange.fr:25
I had the same problem in web hosting https://cloud.ionos.fr

I received "Undelivered Mail Returned to Sender"
host gmail.com[74.125.133.26] said: 550
"550-5.7.26 Unauthenticated email from yahoo.fr is not accepted due to domain's\r\n550-5.7.26 DMARC policy.

Do you have the same problem ?

Bests regards

**Comment by chyshyya** [comment, marked as answer]
this kind of error are throwed when using email on domain but domain doesn't have spf domain record. I think your email address are on domain orange.fr but from answer your provider yahoo.fr, but domain orange.fr hasn't correct spf domain record, so gmail think it unsecure and/or spam.

**Reply by gourdina** [reply]
Thanks chyshyya,
exactly as you say, this problem is fixed
